doc/source/python_reference.rst

```diff
@@ -679,26 +681,40 @@ Time-Frequency
 
 .. currentmodule:: mne.time_frequency
 
+Classes:
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   AverageTFR
+
+Functions that operate on mne-python objects:
+
 .. autosummary::
    :toctree: generated/
    :template: function.rst
 
-   ar_raw
-   compute_raw_psd
-   compute_epochs_psd
-   cwt_morlet
+   csd_epochs
+   psd_raw
+   psd_epochs
    iir_filter_raw
-   morlet
    tfr_morlet
    tfr_multitaper
+   tfr_stockwell
+   read_tfrs
+   write_tfrs
+
+Functions that operate on ``np.ndarray`` objects:
+
+   cwt_morlet
+   dpss_windows
+   morlet
+   multitaper_psd
    single_trial_power
-   yule_walker
    stft
    istft
    stftfreq
-   tfr_stockwell
-   write_tfrs
-   read_tfrs
```

Comment:
Not anymore. The only one used in examples was `ar_raw` (not `yule_walker`), and as far as I could tell all it did was exactly what `iir_filter_raw` did (compute `ar_raw` average AR coeffs, return IIR coeffs), so I replaced it with that, which seemed to make more sense.